src/components/Insight/InsightCardSmall.js

```diff
@@ -22,8 +21,8 @@ const InsightCard = ({
         {title}
       </Link>
       <div className={styles.meta}>
-        <div className={styles.date}>
-          {moment(createdAt).format('MMM D, YYYY')}
+        <div className={styles.username} title={user.username}>
+          {user.username}
```

Comment:
Should be a link to the insights filtered by the author, e.g: `/insights/users/7`

Reply:
done